### ISSUE TITLE
MobileNet does not return activations as commented

### ISSUE THREAD

**Issue by dsmilkov** [tensorflow/tfjs#64]
_From @jaxball on March 28, 2018 17:18_

`models/squeezenet` has a `predictWithActivation` function that returns both named activation and pre-softmax logits. `models/mobilenet` only has a `predict` function which returns a 1D tensor, but its function comment appears identical to that for SqueezeNet, stating that 
>`predict(input: dl.Tensor3D): dl.Tensor1D`
Infer through MobileNet, assumes variables have been loaded. This does standard ImageNet pre-processing before inferring through the model. This method returns named activations as well as pre-softmax logits. 

Can someone verify correctness of the comment and shed light on how to get the activations from MobileNet? 

_Copied from original issue: tensorflow/tfjs-core#907_

**Comment by tafsiri** [comment]
I'm thinking this is out of date with the new tfjs-models code? [That code](https://github.com/tensorflow/tfjs-models/blob/master/mobilenet/index.ts) has an infer and classify method and no predict, and also has a param for internal activations.

closing for now, we can re-open if needed.
